pyccl/core.py

```diff
@@ -276,252 +296,219 @@ def __getitem__(self, key):
         except AttributeError:
             raise KeyError("Parameter '%s' not recognized." % key)
         return val
-    
+
     def __setitem__(self, key, val):
         """Set parameter values by name.
 
         """
         raise NotImplementedError("Parameters objects are immutable; create a "
                                   "new Parameters() instance instead.")
-        """
-        try:
```

Comment:
Where has this gone? Or was it unnecessary?

Reply:
It was commented code.